client/src/components/Button.js

```diff
@@ -1,21 +1,42 @@
+import React from 'react';
 import styled from '@emotion/styled';
+import PropTypes from 'prop-types';
 
-const Button = styled.button`
+const Buttons = styled.button`
   background-color: #8645ff;
   color: white;
-  padding: 12px;
+  padding: 0.7rem;
   border: none;
   border-radius: 20px;
   box-shadow: 0px 10px 15px rgba(0, 0, 0, 16%);
   text-transform: uppercase;
-  font-size: 14px;
-  font-family: Oswald, sans-serif;
-  font-weight: bold;
+  font: bold 0.8rem 'Oswald', sans-serif;
+
   cursor: pointer;
 
   &:active {
     transform: translateY(5px) scale(1.1);
```

Comment:
The button looks great but I'd expect it to rather decrease than increase it's size when clicking it. To make it feel like i'm pushing something.
just a UI/UX thought :-)

Reply:
I don't know what I was thinking 😅I agree it doesn't make much sense... Thanks for your advice, I changed it and it's much better this way. 👌🏼